cmake/stm32_gcc.cmake

```
@@ -1,3 +1,7 @@
+if(${CMAKE_VERSION} VERSION_LESS "3.16.0") 
+    message(WARNING "Current CMake verision is ${CMAKE_VERSION}. stm32-cmake requires CMake 3.16 or greater")

```

Funny, I do this each time also :)

```suggestion
    message(WARNING "Current CMake version is ${CMAKE_VERSION}. stm32-cmake requires CMake 3.16 or greater")

```